lib/lambda/cloudwatch-custom-widget/readers.mts

```diff
@@ -1138,6 +1138,9 @@ interface Node {
   log_server_config: {
     storage_state: string;
   };
+  worker_config: {
```

Comment:
Should we consider it an optional field given that it wasn't always present? It just means that youll need to use ? in more places but it avoids a potential runtime error